client.go

```diff
@@ -610,6 +612,23 @@ func (client *NativeClient) Shell(sin io.Reader, sout, serr io.Writer, args ...s
 		return err
 	}
 
+	// this sends keepalive packets every 30 seconds for a period of SSHKeepAliveTimeout
+	go func() {
+		timeout := time.After(SSHKeepAliveTimeout)
+		tick := time.Tick(30 * time.Second)
```

Comment:
I suggest making the interval a global constant SSHKeepAliveInterval and maybe set it to 1 minute instead of 30 sec.

Reply:
Done. Thanks!